pages/thank-you.jsx

```diff
@@ -32,7 +32,7 @@ var ThankYou = React.createClass({
                 </h2>
               </div>
             </div>
-            {this.props.params && ['de', 'en-US'].indexOf(locale) !== -1 ? <Signup language={locale}/> : <Social language={locale}/> }
+            {this.props.params && ['de', 'en-US'].indexOf(locale) !== -1 ? <Signup locales={this.props.locales}/> : <Social language={locale}/> }
```

Comment:
Yeah, if and when social becomes an issue and needs to include the form mixin, we can update, but if not, it doesn't really bother me as is.